app/views/crosswalk_issues/show_partial.html.erb

```diff
@@ -55,7 +55,7 @@
     </div>
     <div class="row row-space-12">
       <div class="col-xs-12">
-        <%= form_with(url: "/crosswalk_issues/partials", method: "post") do %>
+        <%= form_with(url: crosswalk_issues_resolve_partial_path, method: "post") do %>
```

Comment:
`method: "post"` is superfluous, but it doesn't hurt anything.  `form_with` defaults the method to `post`.